examples/css/core.css

```diff
@@ -76,18 +76,18 @@ form[action^="https://codepen.io/"] button {
   background-image: linear-gradient(to bottom, hsl(216, 82%, 53%), hsl(216, 82%, 47%));
 }
 
-form[action^="https://codepen.io/"] button:hover {
+.example-header button:hover {
   border-color: hsl(213, 71%, 29%);
   background-color: hsl(216, 82%, 31%);
   background-image: linear-gradient(to bottom, hsl(216, 82%, 33%), hsl(216, 82%, 27%));
   cursor: default;
 }
 
-form[action^="https://codepen.io/"] button:focus {
+.example-header button:focus {
   outline: none;
```

Comment:
For some reason I'm still seeing the browser outline on the button itself on Firefox.

Reply:
I can solve this with a vendor prefixed property:

```
.example-header button::-moz-focus-inner {
  border: 0;
}
```

 But I feel like that would make this more convoluted than it needs to be. Thoughts?

Reply:
Maybe we want developers to see that Firefox does this and maybe they'll be convinced to let it be.

The ones who just can't stand it in their own code go to StackOverflow to see hundreds of ways to kill focus outlines. Let them at least do that much leg-work.